src/components/InstancesList/InstancesList.js

```diff
@@ -971,7 +979,49 @@ class InstancesList extends React.Component {
       },
       'contributor': r => {
         if (r?.totalRecords) {
-          return getFullMatchRecord(r?.name, r.isAnchor);
+          const fullMuchRecord = getFullMatchRecord(r.name, r.isAnchor);
+          const isBrowseContributors = this.getQIndexFromParams() === browseModeOptions.CONTRIBUTORS;
+
+          if (isBrowseContributors && r.authorityId) {
+            return (
+              <>
+                <Tooltip
+                  id="marc-authority-tooltip"
+                  text={intl.formatMessage({ id: 'ui-inventory.linkedToMarcAuthority' })}
+                >
+                  {({ ref, ariaIds }) => {
+                    const url = `marc-authorities/authorities/${r.authorityId}?segment=search`;
+
+                    return (
+                      <span
```

Comment:
Can we use a `<TextLink>` from `stripes-components` here?

Reply:
Oh, actually, `<IconButton>` has a `to` prop that turns in into a link. Perhaps we can use that too?

Reply:
@Dmytro-Melnyshyn oh because a row is a `<a>` element itself, got it. Then it's ok